### PR TITLE
Charging functionality

### PR DESCRIPTION
**Pull request by MakkaAkhrieva**
## what

- set stations from malanka url to db with per day update
-implement charging functionality(front part)

### CODE REVIEW

File: client/src/components/MapContainer/MapContainer.js [hunk 1]
```diff
@@ -25,12 +26,60 @@ const MapContainer = () => {
...
+      name: station.name,
+      location: { lat: station.latitude, lng: station.longitude },
+      address: `ул. ${station.street} ${station.house}, ${station.city} , Беларусь`,
+      plugType: station.connectors[0].connectorName,
+    };
+  };
+
+  useEffect(() => {
+    store.getStations();
+    /* document.cookie = "updateData=false; max-age=0"; */
```
Comment (shumak-yaroslav) [inline client/src/components/MapContainer/MapContainer.js]: delete

File: client/src/components/Timer/Timer.js [hunk 2]
```diff
@@ -0,0 +1,47 @@
...
+import { Context } from "../../index.js";
+
+export const MyStopwatch = () => {
+  const { store } = useContext(Context);
+  const { seconds, minutes, hours, days, isRunning, start, pause, reset } =
+    useStopwatch({ autoStart: false });
+  const [time, setTime] = useState(null);
+  const [price, setPrice] = useState(0);
+
+  const end = () => {
```
Comment (shumak-yaroslav) [inline client/src/components/Timer/Timer.js]: useCallback

File: client/src/components/Timer/Timer.js [hunk 3]
```diff
@@ -0,0 +1,47 @@
...
+    pause();
+    setTime(+minutes + +(seconds / 60).toFixed(2));
+  };
+
+  const resetTime = () => {
```
Comment (shumak-yaroslav) [inline client/src/components/Timer/Timer.js]: useCallback

File: server/router/index.js [hunk 4]
```diff
@@ -18,10 +18,11 @@ router.post("/logout", userController.logout);
 router.get("/activate/:link", userController.activate);
 router.get("/refresh", userController.refresh);
 router.get("/stations", stationController.getStations);
+router.get("/station/:id", authMiddleware, stationController.getStation);
 router.post(
   "/station",
-  authMiddleware,
-  adminMiddleware,
+  /*   authMiddleware,
```
Comment (shumak-yaroslav) [inline server/router/index.js:25]: delete

File: client/src/components/MapInfoMenu/MapInfoMenu.js [hunk 5]
```diff
@@ -44,6 +45,11 @@ const MapInfoMenu = ({
     setExpanded(!expanded);
   };
 
+  useEffect(() => {
+    localStorage.setItem("stationid", selectedMarker.id);
+    console.log("selectedid", selectedMarker.id);
```
Comment (shumak-yaroslav) [inline client/src/components/MapInfoMenu/MapInfoMenu.js]: delete

File: server/router/index.js (hunk 4 above)
Comment (MakkaAkhrieva) [reply server/router/index.js:25]: i need it)

**Pull request merged**